vulcanai/models/basenetwork.py

```diff
@@ -651,9 +652,11 @@ def fit(self, train_loader, val_loader, epochs,
                 else:
                     save_path = save_path + '/' + self.name + '_'
                 save_path = get_save_path(save_path, vis_type='train')
-            iterator = trange(epochs, desc='Epoch: ')
+            # iterator = trange(epochs, desc='Epoch: ')
```

Comment:
doesn't this remove the progress bar at the epoch level?

Reply:
The problem was the overlapping progress bars, that's why i removed the outer one

Reply:
but you still need to print the epoch progress bar. Nested progress bars are what we are going for so they just need to be cleaned up in implementation.